torch/_functorch/aot_autograd.py

```diff
@@ -1214,7 +1211,9 @@ def aot_module_simplified(
 
             stack.enter_context(compiled_autograd._disable())
 
-            compiler_fn, flat_fn, dup_fake_flat_args, aot_config, fw_metadata = (
+            flat_fn = functional_call
```

Comment:
where's `flat_fn` being used in this file after this point?

Reply:
It's used line 1228 below? I'm not sure what you actually mean here lol.